src/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+import './index.scss';
```

Comment:
* rename src/sass put it in assets and then rename sass to styles
* remove this file src/index.scss
* only import the main style file from src/assets/styles here
* add the other folders and of styles for the 7 to 1 style

Reply:
Made changes